tests/cupyx_tests/scipy_tests/stats_tests/test_stats_py.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+
+import numpy
+
+import cupy
+
+from cupy import testing
+
+import cupyx
+import cupyx.scipy.stats  # NOQA
+
+import scipy.stats  # NOQA
+
+
+atol = {'default': 1e-6, cupy.float64: 1e-14}
+rtol = {'default': 1e-6, cupy.float64: 1e-14}
+
+
+@testing.with_requires('scipy')
+class TestZmap:
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_allclose(scipy_name='scp')
+    def test_zmap_1dim(self, xp, scp, dtype):
+        x = testing.shaped_random((5,), xp, dtype=dtype)
+        y = testing.shaped_random((4,), xp, dtype=dtype)
```

Comment:
Would you add tests where `compare` arrays have the same dimension with `scores` as scipy's assumes as such?

It also seems true that scipy's accept `compare` with the different dimension without seeing any exceptions or warnings. So I think keeping the current tests is okay. (only multi-dim would be enough)

Reply:
Added test. Thanks for your input :)